frontend/pages/DetailedViewPage.js

```diff
@@ -77,7 +99,7 @@ export default class DetailedViewPage extends Component {
                     </View>
                     <View style={styles.buttonGroup}>
                         <Button containerStyle={styles.dialogButton} title="CANCEL" onPress={this.toggleChangePriceOverlay} />
-                        <Button containerStyle={styles.dialogButton} title="SUBMIT" onPress={this.addNewList} />
+                        <Button containerStyle={styles.dialogButton} title="SUBMIT" onPress={this.changePrice} />
                     </View>
                 </View>
             </Overlay>
```

Comment:
In line 136 of this file, it might be better to change 
```
        flexDirection: "row"

```
to 
```
        flexDirection: 'row'

```

Reply:
Nice catch, we will make our quotation styling consistent across the file.